src/emails/components/EmailHeader.tsx

```diff
@@ -48,3 +48,12 @@ export const EmailHeader = (props: Props) => {
     </mj-section>
   );
 };
+
+export const getUnstyledEmailHeader = (props: Props) => {
+  const l10n = props.l10n;
+
+  return `
```

Comment:
Suggestion: The linebreak behind directly after the backtick causes the first line to be empty. In case you’d like to avoid this you could escape it:

```suggestion
  return `\
```

Reply:
TIL, thanks! 11c9d05de